demos/python_demos/common/monitors_extension/monitors_extension.cpp

```diff
@@ -13,11 +13,12 @@
 
 struct PresenterObject {
     PyObject_HEAD
-    std::unique_ptr<Presenter> _presenter;
+    Presenter *_presenter;
 };
 
 namespace {
 void presenter_dealloc(PresenterObject *self) {
+    delete self->_presenter;
     Py_TYPE(self)->tp_free(self);
```

Comment:
You also need to decref the type, since it's now a heap type (see https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_dealloc).

Reply:
Fixed